tests/dummy/app/templates/docs/transitions.md

```diff
@@ -0,0 +1,49 @@
+# Transitions
+
+Transitions assign each sprite to a motion. A transition tells sprite categories how to animate from one state to another. For example, a transition can assign the `fade` motion to all `insertedSprites`. Each time a sprite gets added to a list, it would fade into the list as that sprite went from its initial to its final state. 
+
+Transitions are interruptible and they run every time the state of a sprite changes. Therefore, they must be implemented as generator functions. Also, this means that the state of a sprite can change during a transition.
+
+`fade` and `moveOver` are built-in transitions. To use the built in transitions, import `fade` or `moveOver` and then pass them into your template.
+
+In this demonstration, the message fades in or it fades out as it changes states. Each click of the button makes the state of the message change, causing the transition to run.
+
+### Fade
+
+{{#docs-demo as |demo|}}
+    {{#demo.example name="fade"}}
+      {{transitions-fade}}
+    {{/demo.example}}
+
+    {{demo.snippet 'transitions-fade-snippet.hbs' label='transitions-fade.hbs'}}
+    {{demo.snippet 'transitions-fade-snippet.js' label='transitions-fade.js'}}
+    {{demo.snippet 'transitions-snippet.css'}}
+{{/docs-demo}}
+
+
+`moveOver` comes with four options: `toLeft`, `toRight`, `toUp`, and `toDown`. `toLeft` and `toRight` are demonstrated here, using `rules` to compare two messages and choose a transition. For more on `rules` see [Rules for Data-Dependent Animations](../docs/rules/).
+
+### moveOver
+
+{{#docs-demo as |demo|}}
+  {{#demo.example name="moveover"}}
+    {{transitions-moveover}}
+  {{/demo.example}}
+
+  {{demo.snippet 'transitions-moveover-snippet.hbs' label='transitions-moveover.hbs'}}
+  {{demo.snippet 'transitions-moveover-snippet.js' label='transitions-moveover.js'}}
+{{/docs-demo}}
+
+
+You can also create your own transitions. `slideFromSide` is a custom transition that assigns motions to `insertedSprites`, `keptSprites`, and `removedSprites`. The message will slide to and from the side when the button is selected. 
```

Comment:
If there are any docs about creating your own transitions, they would be good to add here